Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -301,7 +458,7 @@ void TextureMapperLayer::sortByZOrder(Vector<TextureMapperLayer* >& array)
         });
 }
 
-void TextureMapperLayer::paintSelfAndChildren(TextureMapperPaintOptions& options)
+void TextureMapperLayer::paintSelfAndChildren(TextureMapperPaintOptions& options, bool includesFlattenedChildren)
```

Comment:
If it's generally useful, I'd rather add this to the TextureMapperPaintOptions, if not, I'd add an `enum class ShouldIncludeFlattenedChildren : bool { Yes, No }` to the header, and use that instead of a boolean - which is always harder to read on the call site, as it's not descriptive.

Reply:
Refactored code and this parameter is now removed